app/src/main/java/com/rokt/roktdemo/ui/layouts/ScanQrViewModel.kt

```diff
@@ -54,18 +58,45 @@ class ScanQrViewModel @Inject constructor(
     fun executePreview(embeddedWidget: WeakReference<Widget>?) {
         val scannedData = _state.value.data?.scannedData
         scannedData?.let { data ->
-            val attributes = hashMapOf(
-                ATTRIBUTE_IS_DEMO to true.toString(),
-                ATTRIBUTE_LAYOUT_ID to data.previewId,
-                ATTRIBUTE_VERSION_ID to data.versionId,
-                ATTRIBUTE_CREATIVE_ID to data.creativeIds.joinToString(separator = ",")
-            )
+            val attributes = buildMap {
+                this[ATTRIBUTE_IS_DEMO] = true.toString()
+                this[ATTRIBUTE_LAYOUT_ID] = data.previewId
+                this[ATTRIBUTE_VERSION_ID] = data.versionId
```

Comment:
These two wouldn't be needed anymore, because we send them via demoConfig

Reply:
Thanks. Done.